t/admin/standalone.spec.ts

```diff
@@ -194,6 +194,86 @@ describe('Admin - Standalone', () => {
   });
 
   describe('Normal', () => {
+    it('validate config (success case with json)', async () => {
+      const resp = await client.post(`${ENDPOINT}/validate`, config1);
+      expect(resp.status).toEqual(200);
+      expect(resp.data).toEqual({
+        message: 'Configuration is valid',
+        valid: true,
+      });
+    });
+
+    it('validate config (success case with yaml)', async () => {
+      const resp = await client.post(`${ENDPOINT}/validate`, YAML.stringify(config1), {
+        headers: { 'Content-Type': 'application/yaml' },
+      });
+      expect(resp.status).toEqual(200);
+      expect(resp.data).toEqual({
+        message: 'Configuration is valid',
+        valid: true,
+      });
+    });
+
+    it('validate config (success case with multiple resources)', async () => {
+      const multiResourceConfig = {
+        routes: [
+          {
+            id: 'r1',
+            uri: '/r1',
+            upstream: {
+              nodes: { '127.0.0.1:1980': 1 },
+              type: 'roundrobin',
+            },
+          },
+          {
+            id: 'r2',
+            uri: '/r2',
+            upstream: {
+              nodes: { '127.0.0.1:1980': 1 },
+              type: 'roundrobin',
+            },
+          },
+        ],
+        services: [
+          {
+            id: 's1',
+            upstream: {
+              nodes: { '127.0.0.1:1980': 1 },
+              type: 'roundrobin',
+            },
+          },
+        ],
+        routes_conf_version: 1,
+        services_conf_version: 1,
+      };
+
+      const resp = await client.post(`${ENDPOINT}/validate`, multiResourceConfig);
+      expect(resp.status).toEqual(200);
+      expect(resp.data).toEqual({
+        message: 'Configuration is valid',
+        valid: true,
+      });
+    });
+
+    it('validate config with consumer credentials', async () => {
+      const resp = await client.post(`${ENDPOINT}/validate`, credential1);
+      expect(resp.status).toEqual(200);
+      expect(resp.data).toEqual({
+        message: 'Configuration is valid',
```

Comment:
This `message` does not need assertion, as checking such constant strings is meaningless. You may switch to the `toMatchObject` API to assert fields other than `message`. Only fields expected to exist in the configuration will be checked.

If no further assertion of the response is required, then only assert the response status code.